mailing_list/tasks.py

```diff
@@ -0,0 +1,57 @@
+"""
+Defines huey tasks to sync mailing list membership from our systems out to
+the listserv service.
+"""
+
+import collections
+import logging
+
+from django.conf import settings
+from huey.djhuey import crontab, db_task, db_periodic_task
+
+from canvas_sdk.exceptions import CanvasAPIError
+
+from mailing_list.models import MailingList
+from mailing_list.listserv_clients.exceptions import ListservApiError
+
+
+logger = logging.getLogger(__name__)
+
+# if we don't supply a default, huey will run it every minute.  let's opt
+# for once an hour instead.
+DEFAULT_LISTSERV_PERIODIC_SYNC_CRONTAB = {'minute': '0'}
+
+
+@db_task()
+def sync_listserv(course_ids=None):
+    _sync_listserv(course_ids)
+
+
+@db_periodic_task(
+    crontab(**getattr(settings, 'LISTSERV_PERIODIC_SYNC_CRONTAB',
+                                DEFAULT_LISTSERV_PERIODIC_SYNC_CRONTAB)))
+def periodic_sync_listserv():
+    _sync_listserv(None)
+
+
+def _sync_listserv(course_ids):
```

Comment:
Can we move the contents of this to a utility function that both this and the management command can call?